package com.cyrillo.ativo.core.dataprovider.excecao;

public class FalhaObterConexaoRepoDataProvExcecao extends Exception {
    public FalhaObterConexaoRepoDataProvExcecao(String s){
        super(s);
    }
}
